tests/h/streamer/websocket_test.py

```diff
@@ -319,12 +315,14 @@ def test_expands_uris_in_uri_filter_with_session(self, expand_uri, socket):
 
         websocket.handle_filter_message(message, session=session)
 
-        uri_filter = socket.filter.filter["clauses"][0]
-        uri_values = uri_filter["value"]
-        assert len(uri_values) == 3
-        assert "http://example.com" in uri_values
-        assert "http://example.com/alter" in uri_values
-        assert "http://example.com/print" in uri_values
+        # Use Any.list to do order insensitive matching
+        expected_values = Any.list.containing(expand_uri.return_value).only()
```

Comment:
I presume `.only()` means that the matching will fail if the actual list contains values not in `expand_uri.return_value` in addition to all the values in `expand_uri.return_value`?

Reply:
Yep that's it. You can also call `in_order()`, but by that point you can just compare with a fixed list